src/LiquidityPool.sol

```diff
@@ -461,6 +462,20 @@ contract LiquidityPool is Initializable, OwnableUpgradeable, UUPSUpgradeable, IL
 
         emit Rebase(getTotalPooledEther(), eETH.totalShares());
     }
+    /// @notice pay protocol fees including 5% to treaury, 5% to node operator and ethfund bnft holders
+    /// @param _protocolFees The amount of protocol fees to pay in ether
+    function payProtocolFees(uint128 _protocolFees) external {
+        if (msg.sender != address(etherFiAdminContract)) revert IncorrectCaller();   
```

Comment:
Is it necessary though the caller is being checekd on `depositToRecipient` function

Reply:
not necessary just more clear and its a little stronger because if I remove it liquifier contract can call method.